### ISSUE TITLE
release 1.4.0

### ISSUE THREAD

**Issue by d0kt0r** [B3none/retakes-instadefuse#21]
In the latest release, after installing the plugin, the standard voting from the server disappears, the cards themselves change

**Comment by b3none** [comment]
Hey @d0kt0r, could you please provide some evidence of this behaviour occurring? :)

**Comment by zahti** [comment]
@b3none ,
Could you please put the **retakes-instadefuse.smx** compiled plugin in the latest release? Looks like it is in 1.2.0 but missing in 1.4.0. I don't know how to manually compile it otherwise I would.

**Comment by b3none** [comment]
> @b3none ,
> Could you please put the **retakes-instadefuse.smx** compiled plugin in the latest release? Looks like it is in 1.2.0 but missing in 1.4.0. I don't know how to manually compile it otherwise I would.

Well spotted! I'll add it now.

**Comment by zahti** [comment]
@b3none Thanks!!
Any good guides on that? I looked at it but not familiar with how to compile the SM plugins. How is it different than this? https://www.sourcemod.net/compiler.php

**Comment by zahti** [comment]
@b3none Also, how do I debug the plugin not working at all? I got your autoplant plugin working with no issues (amazing btw!), but the instadefuse doesn't seem to do anything. I have placed the 1.4.0 compiled release directly in the /addons/sourcemod/plugins folder.

![2020-03-26 11_31_10-5e7bd101988b2559ec104cff@qyburn dathost net - FileZilla](https://user-images.githubusercontent.com/38710634/77677599-8acfd100-6f55-11ea-94ef-5a8fefe917f5.png)
